gallery/src/components/NxRadio/NxRadioPage.tsx

```diff
@@ -157,6 +157,7 @@ const NxRadioPage = () =>
                         liveExample={NxRadioDisabledExample}>
       This example shows radios that are disabled.
       Disabled radios can either be checked or unchecked.
+      The example also demonstrates the use of the isRequired flag to remove the "Optional" indicator.
```

Comment:
Another instance of this inaccurate sentence

Reply:
Removed here: [6ed96b0](https://github.com/sonatype/sonatype-react-shared-components/pull/735/commits/6ed96b0aa719a655327e16eb493dadc16752ee2e)